components/Input/Input.tsx

```diff
@@ -0,0 +1,61 @@
+import { InputHTMLAttributes } from "react";
+
+interface Props
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    "id" | "type" | "value" | "onChange" | "name" | "required" | "className"
+  > {
+  error?: boolean;
+  label: string;
+  helperText: string;
+}
+
+const Input = ({
+  id,
+  error = false,
+  value,
+  onChange,
+  type,
+  name,
+  label,
+  required,
+  className,
+  helperText,
+}: Props) => (
+  <div className="flex flex-col">
+    <label
+      htmlFor={name}
```

Comment:
`htmlFor` (o `for`) siempre deberia recibir el `id` del `input` al que esta vinculado